### DISCUSSION TITLE
How to close the root view and shutdown the whole app?

### DISCUSSION THREAD

**Discussion by mwdpb** [canton7/Stylet#248] [Q&A] [answered]
I have a "Close" button on the root view GUI and upon clicking it, the whole app should shutdown.  What code should I put inside this command callback?  Forgive me if it's a trivial question, but I've gone thru the Wiki as well as the Discussion forum but couldn't find the solution.  Thanks in advance.

**Comment by canton7** [comment, marked as answer]
[RequestClose](https://github.com/canton7/Stylet/wiki/Screens-and-Conductors) should do it!

**Reply by mwdpb** [reply]
I did notice that in the Wiki documentation, but I don't have access to that method, what super-class should I inherit the root view model from?  Currently I just inherit the "PropertyChangedBase".

![image](https://user-images.githubusercontent.com/55845355/116736669-33a0f900-a9be-11eb-936b-0f7998f0e25d.png)

**Reply by canton7** [reply]
From Screen

**Reply by mwdpb** [reply]
Works great!  Thank you!
